src/System.Net.Requests/tests/HttpWebRequestTest.cs

```diff
@@ -188,6 +188,25 @@ public void ContentType_SetThenGetEmptyValue_ExpectNull(Uri remoteServer)
             });
         }
 
+        [Fact]
+        public async Task HttpWebRequest_SetHostHeader_ContainsPortNumber()
+        {
+            await LoopbackServer.CreateServerAsync(async (server, uri) =>
+            {
+                HttpWebRequest request = WebRequest.CreateHttp(uri);
+                string host = uri.Host + ":" + uri.Port;
+                request.Host = host;
+                Task<WebResponse> getResponse = request.GetResponseAsync();
```

Comment:
There isn't any "await getResponse" on this task.  So, that leaves a "runaway" async I/O happening.  In fact, since this loopback server isn't even sending out a valid HTTP response, doing an await will hang.  

On a busy CI test machine leaving async i/o hanging like this could cause problems.  It would be better if this test could finish the http loopback server response, send out a small proper response (i.e. 200 OK etc.) and then await the 'getResponse' task.

Reply:
Will fix.

Reply:
Will fix.